explorer/src/constants/routes.ts

```diff
@@ -42,14 +41,14 @@ export const EXTERNAL_ROUTES = {
   operatorDocs:
     'https://docs.subspace.network/docs/farming-&-staking/staking/operators/register-operator',
   social: {
-    twitter: 'https://x.com/AutonomysNet',
+    twitter: 'https://twitter.com/NetworkSubspace',
     discord: 'https://discord.gg/subspace-network',
     telegram: 'https://t.me/subspace_network',
     github: 'https://github.com/subspace',
     reddit: 'https://www.reddit.com/r/sub',
     medium: 'https://medium.com/subspace-network',
     youtube: 'https://www.youtube.com/channel/UCojYRCZOtVTJHJXivOYJzeQ',
-    linkedin: 'https://www.linkedin.com/company/autonomys/',
+    linkedin: 'https://www.linkedin.com/company/subspace-blockchain/',
```

Comment:
Were these changes intended?

Reply:
Good catch, no, this revert is not intentional 😭